docs/options.md

```diff
@@ -293,12 +293,16 @@ or
 
 _First available in v1.8.0_
 
-Prettier can insert a special @format marker at the top of files specifying that the file has been formatted with prettier. This works well when used in tandem with the `--require-pragma` option. If there is already a docblock at the top of the file then this option will add a newline to it with the @format marker.
+Prettier can insert a special `@format` marker at the top of files specifying that the file has been formatted with Prettier. This works well when used in tandem with the `--require-pragma` option. If there is already a docblock at the top of the file then this option will add a newline to it with the `@format` marker.
```

Comment:
Seems missing mention of `@prettier` marker.

Reply:
This is because `--insert-pragma` inserts specifically `@format`.

Reply:
Oh, `@prettier` mentioned this in `--require-pragma`